### PR TITLE
Add frontend to list the products ordered by a user

### PR DESCRIPTION
**Pull request by arjavibahety**
image

### CODE REVIEW

File: webproject/src/main/java/com/google/sps/servlets/MyOrderServlet.java [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+import java.net.HttpURLConnection;
+import java.net.URL;
+import java.io.OutputStream;
+import java.io.BufferedReader;
+import java.io.InputStreamReader;
+
+@WebServlet("/myOrder")
+public class MyOrderServlet extends HttpServlet {
+  // TODO
+}
```
Comment (kevinluvian) [inline webproject/src/main/java/com/google/sps/servlets/MyOrderServlet.java]: newline

File: webproject/src/main/webapp/js/roomMyOrder.js [hunk 2]
```diff
@@ -0,0 +1,159 @@
...
+    </td>
+    <td>
+    <input type="number" class="form-control" id="newProductUnitPrice" required />
+    </td>
+    <td><input type="submit" class="btn btn-add" value="Add" />
+    </td>
+    <td></td>
+    </tr>
+    </form>`;
+}
```
Comment (kevinluvian) [inline webproject/src/main/webapp/js/roomMyOrder.js]: newline

File: webproject/src/main/java/com/google/sps/servlets/MyOrderServlet.java [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+import javax.servlet.http.HttpServletResponse;
+import javax.servlet.annotation.WebServlet;
+import java.io.IOException;
+import java.net.HttpURLConnection;
+import java.net.URL;
+import java.io.OutputStream;
+import java.io.BufferedReader;
+import java.io.InputStreamReader;
+
+@WebServlet("/myOrder")
```
Comment (kevinluvian) [inline webproject/src/main/java/com/google/sps/servlets/MyOrderServlet.java:17]: I think this should have roomId in the path. but we can modify later
FYI @bxtsang

**Review by kevinluvian — CHANGES_REQUESTED** [review]

File: webproject/src/main/java/com/google/sps/servlets/MyOrderServlet.java (hunk 1 above)
Comment (arjavibahety) [reply webproject/src/main/java/com/google/sps/servlets/MyOrderServlet.java]: Sorry about it! The issue was my VS Code's formatter >.<

**Review by kevinluvian — APPROVED** [review]

**Pull request merged**